src/main/java/com/rocketden/main/dto/game/GameDto.java

```diff
@@ -3,14 +3,15 @@
 import lombok.Getter;
 import lombok.Setter;
 
-import java.util.Map;
+import java.util.ArrayList;
+import java.util.List;
 
 import com.rocketden.main.dto.room.RoomDto;
 
 @Getter
 @Setter
 public class GameDto {
-    private RoomDto roomDto;
-    private Map<String, PlayerDto> playerMap;
```

Comment:
I'm not sure how maps translate to objects on the frontend, and I figured we won't really need to do any intense quick-access stuff on the game page besides just listing all the players, hence I thought list might be better here - open to thoughts as always tho

Reply:
Ok that makes sense -- initially I thought you were changing the `Map` design on the `Game` object instead of `GameDto`, which I think would've been a less valid choice, but this modification makes sense.